snuba/query/parser/expressions.py

```diff
@@ -77,7 +77,7 @@
 )
 
 
-class ClickhouseVisitor(NodeVisitor):
+class ClickhouseVisitor(NodeVisitor):  # type: ignore
```

Comment:
What was the issue here ?

Reply:
NodeVisitor has type Any and mypy warns on extending something with type: Any.

Reply:
Mind adding a comment to explain ?